### PR TITLE
CMS: HI validated runs

### LINKED ISSUE #2908: CMS: create validated run records for HI
Create 6 records as in https://github.com/cernopendata/data-curation/issues/85
Update 24.11: Added 2 more for 2.76 TeV pp (https://github.com/cernopendata/data-curation/issues/85#issuecomment-733128773)

Files are in https://cernbox.cern.ch/index.php/s/D236HWZGuMOnSZB

can be placed in eos/opendata/cms/validated-runs/hi/2010/ (150436...) and eos/opendata/cms/validated-runs/hi/2011/

recids: 14202-14207 + 14208-14209

### PR DESCRIPTION
**Pull request by katilp**
(closes #2908 and https://github.com/cernopendata/data-curation/issues/85)

Needs:
- files upload from  https://cernbox.cern.ch/index.php/s/D236HWZGuMOnSZB

### CODE REVIEW

File: cernopendata/modules/fixtures/data/records/cms-validated-runs-HIRun1.json [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+        "txt"
+      ]
+    },
+    "experiment": "CMS",
+    "publisher": "CERN Open Data Portal",
+    "recid": "14202",
+    "run_period": [
+      "HIRun2010"
+    ],
+    "title": "CMS list of validated runs Cert_150436-152957_HI7TeV_StreamExpress_Collisions10_JSON_v2.txt",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-validated-runs-HIRun1.json:30]: The title field says `Cert_150436-152957` but the files themselves are called `Cert_150436-151238`, which one is correct?

Comment (katilp) [reply cernopendata/modules/fixtures/data/records/cms-validated-runs-HIRun1.json:30]: Thanks for pointing that out! The files are wrong, the title is correct. I'll upload the new ones right now.

Comment (katilp) [reply cernopendata/modules/fixtures/data/records/cms-validated-runs-HIRun1.json:30]: The two correct files are now in https://cernbox.cern.ch/index.php/s/D236HWZGuMOnSZB. This affects both records having `Cert_150436-152957`.

Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/cms-validated-runs-HIRun1.json:30]: Thanks, all updated, repushing inside the same PR, you'll be able to verify everything later on DEV...

**Review by tiborsimko — APPROVED** [review]

**Pull request merged**